dll/shellext/stobject/lang/fr-FR.rc

```diff
@@ -16,26 +16,26 @@ STRINGTABLE
 BEGIN
     //Power related strings
     IDS_PWR_PROPERTIES        "&Modifier les paramètres d'alimentation"
-    IDS_PWR_METER             "&Open Power Meter"
+    IDS_PWR_METER             "&Ouvrir le gestionnaire d'énergie"
     IDS_PWR_RUN               "shell32.dll,Control_RunDLL PowerCfg.cpl"
-    IDS_PWR_PERCENT_REMAINING "%u%% remaining"
-    IDS_PWR_CHARGING          "%u%% and charging"
-    IDS_PWR_UNKNOWN_REMAINING "Unknown remaining"
-    IDS_PWR_AC "On AC power"
-    IDS_PWR_HOURS_REMAINING "%1!u!:%2!02u! hours (%3!u!%%) remaining"
-    IDS_PWR_MINUTES_REMAINING "%1!u! min (%2!u!%%) remaining"
+    IDS_PWR_PERCENT_REMAINING "%u%% restants"
+    IDS_PWR_CHARGING          "%u%% et en charge"
+    IDS_PWR_UNKNOWN_REMAINING "Reste inconnu"
+    IDS_PWR_AC "Sur secteur"
+    IDS_PWR_HOURS_REMAINING "%1!u!:%2!02u! heures (%3!u!%%) restantes"
+    IDS_PWR_MINUTES_REMAINING "%1!u! mins (%2!u!%%) restantes"
 
     //Hotplug related strings
-    IDS_HOTPLUG_REMOVE_1 "Safely Remove Hardware"
-    IDS_HOTPLUG_REMOVE_2 "&Safely Remove Hardware"
-    IDS_HOTPLUG_REMOVE_3 "Safely remove %s"
+    IDS_HOTPLUG_REMOVE_1 "Retirer le matériel en sécurité"
+    IDS_HOTPLUG_REMOVE_2 "&Retirer le matériel en sécurité"
```

Comment:
Maybe s/matériel/périphérique/ (as done below)

Reply:
I align with "Périphérique" in all places for the sake of consistency with your other comments.